stl/src/msvcp_atomic_wait.src

```diff
@@ -6,14 +6,17 @@
 LIBRARY LIBRARYNAME
 
 EXPORTS
-    __std_atomic_wait_get_deadline
-    __std_atomic_wait_get_remaining_timeout
+    __std_atomic_compare_exchange_128

```

Comment:
@BillyONeal Is there a list of exports that we need to update for the "have the exports changed" build-time test? (I think that might still be MSVC-internal.) Are we running such a check against the atomic_wait satellite yet?